airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGeneratorIntegrationTest.java

```diff
@@ -181,6 +187,30 @@ public void testCreateTableIncremental() throws InterruptedException {
     // TODO this should assert partitioning/clustering configs
   }
 
+  @Test
+  public void testCreateTableInOtherRegion() throws InterruptedException {
+    final StreamConfig stream = incrementalDedupStreamConfig();
+    BigQueryDestinationHandler destinationHandler = new BigQueryDestinationHandler(bq, "asia-east1");
+    // We're creating the dataset in the wrong location in the @BeforeEach block. Explicitly delete it.
+    bq.getDataset(testDataset).delete();
+
+    destinationHandler.execute(new BigQuerySqlGenerator("asia-east1").createTable(stream, ""));
+
+    // Empirically, it sometimes takes Bigquery nearly 30 seconds to propagate the dataset's existence.
+    // Give ourselves 2 minutes just in case.
+    for (int i = 0; i < 120; i++) {
```

Comment:
this is dumb, but we're running it concurrently with other tests that take longer (e.g. most TypingDedupingTests testcases need to run two full syncs).